src/main/java/de/abstractolotl/azplace/rest/controller/BoardController.java

```diff
@@ -81,7 +82,14 @@ public PixelInfoView pixel(int canvasId, int x, int y) {
             personId = pixelOwner.getUser().getInsideNetIdentifier();
         }
 
+        int userId = -1;
+        if(!authService.hasRole(pixelOwner.getUser(), "anonymous") ||
```

Comment:
magic values sollten vermieden werden
("anonymous" bzw "admin")

Reply:
Das ist überall anders auch noch so, dass sollte dann in einem generellen Refactor geändert werden

Reply:
https://github.com/Abstractolotl/azplace-rest-backend/issues/70